### ISSUE TITLE
For all investigations - Widgets and animations should respect browser prefers-reduced-motion setting

### ISSUE THREAD

**Issue by jeffinnes** [lsst-epo/investigations-client#455]
Some users have provided feedback that auto-playing widgets and the celebration fireworks are disorienting or not as accessible for folks with things like motion sensitivity or autism.

Animated features should be disabled and we should enhance all of our motion based widgets to detect this setting and start in the paused state.

In CSS this can be done with a media selector like (class selector and rule do not reflect actual code in the app):

```css
@media (prefers-reduced-motion: reduce) {
  .animated-gif {
    display: none;
  }
}
```

In JavaScript/React we can use something like this [React custom hook from Josh Comeau](https://www.joshwcomeau.com/snippets/react-hooks/use-prefers-reduced-motion/).
